src/main/java/org/prebid/server/auction/BidResponseCreator.java

```diff
@@ -163,6 +163,7 @@ private static int validateTruncateAttrChars(int truncateAttrChars) {
                                AuctionContext auctionContext,
                                BidRequestCacheInfo cacheInfo,
                                Map<String, MultiBidConfig> bidderToMultiBids,
+                               List<String> debugWarnings,
```

Comment:
This parameter doesn't look necessary - `debugWarnings` could be obtained from `auctionContext`